trieste/utils/misc.py

```diff
@@ -210,3 +212,18 @@ def map_values(f: Callable[[U], V], mapping: Mapping[K, U]) -> Mapping[K, V]:
         applying ``f`` to each value in ``mapping``.
     """
     return {k: f(u) for k, u in mapping.items()}
+
+
+class timeit:
```

Comment:
Sorry to be a pain, but now that were' implementing timeit as a class, we should really name it TimeIt instead!

Also there should be a class docstring explaining what this is for and how to use it.